File: Data Structures & Algorithms/reverse-integer/submission-2.cpp
class Solution {
public:
    int reverse(int x) {
        bool sign=true;
        if(x<0)
        sign=false;

        long long n=abs((long long)x);
        long long ans=0;
        while(n!=0)
        {
        long long residue=n%10;
        ans=ans*10+residue;
        n=n/10;
        }
        if(sign==true && ans>(long long) INT_MAX)
        return 0;
        else if(sign==false && ans>(long long)INT_MAX+1)
        return 0;
        else if(sign==true)
        return ans;
        else
        return ans*(-1);
        
    }
};

File: Data Structures & Algorithms/top-k-elements-in-list/submission-2.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        mpp={}
        for i in range(len(nums)):
            if nums[i] in mpp:
                mpp[nums[i]]+=1
            else:
                mpp[nums[i]]=1
        v=[]
        for key in mpp:
            v.append([mpp[key],key])
        v.sort(reverse=True)
        ans=[]

        for i in range(k):
            ans.append(v[i][1])
        return ans
        